src/briefcase/integrations/java.py

```diff
@@ -310,8 +310,17 @@ def install(self):
 
     def uninstall(self):
         """Uninstall a JDK."""
-        with self.tools.console.wait_bar("Removing old JDK install..."):
-            if self.tools.host_os == "Darwin":
-                self.tools.shutil.rmtree(self.java_home.parent.parent)
-            else:
-                self.tools.shutil.rmtree(self.java_home)
+        try:
+            with self.tools.console.wait_bar("Removing old JDK install..."):
+                if self.tools.host_os == "Darwin":
+                    self.tools.shutil.rmtree(self.java_home.parent.parent)
+                else:
+                    self.tools.shutil.rmtree(self.java_home)
+        except PermissionError as e:
+            raise BriefcaseCommandError(
+                f"""\
+Permission denied when trying to remove {self.java_home}.
```

Comment:
This message will be incorrect (or at least a little misleading) if you're on macOS (i.e., Darwin).